addon-registry.js

```diff
@@ -130,6 +130,10 @@ class AddonConfigurationRegistry {
     this.packages = {};
     this.customizations = new Map();
 
+    // Theme from a package.json key or from volto.config.js
+    // Programatically via volto.config.js wins
+    this.theme = packageJson.theme || this.voltoConfigJS.theme;
```

Comment:
I might want an addon to provide the theme.

The volto.config.js implies that the project is no longer throw-away, or at least there's a standard "distribution-like" volto.config.js that I use in all my projects.

Reply:
I agree that would be desirable, but the current state is that you need to define the enabled `addons` list in `volto.config.js` or in `package.json` anyways, so adding the `theme` is nothing that we "loose"...

In fact, these are the only things that you need to "save" in order of a project to be expendable. Same applies for the dockerized approach... as long as you have a `volto.config.js` that you can forward to your expendable project you are safe.

Reply:
I would add an option to define a theme name from an environment var. We have it already for addons iirc, so it would be a nice complement, and a very very nice addition for docker users :)